tools/peptideshaker/peptide_shaker.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="peptide_shaker" name="Peptide Shaker" version="1.16.30">
+<tool id="peptide_shaker" name="Peptide Shaker" version="@PEPTIDESHAKER_VERSION@">
```

Comment:
should the peptide-shaker version also be used in the requirements?

Reply:
oh. sure. :)